config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml

```diff
@@ -147,43 +145,36 @@ spec:
         - internalapi
       manilaScheduler:
         replicas: 1
-        containerImage: quay.io/tripleozedcentos9/openstack-manila-scheduler:current-tripleo
         networkAttachments:
         - internalapi
       manilaShares:
         share1:
-          containerImage: quay.io/tripleozedcentos9/openstack-manila-share:current-tripleo
           replicas: 1
           networkAttachments:
           - internalapi
   ovn:
     template:
       ovnDBCluster:
         ovndbcluster-nb:
-          containerImage: quay.io/tripleozedcentos9/openstack-ovn-nb-db-server:current-tripleo
           dbType: NB
           storageRequest: 10G
           networkAttachment: internalapi
         ovndbcluster-sb:
-          containerImage: quay.io/tripleozedcentos9/openstack-ovn-sb-db-server:current-tripleo
           dbType: SB
           storageRequest: 10G
           networkAttachment: internalapi
       ovnNorthd:
-        containerImage: quay.io/tripleozedcentos9/openstack-ovn-northd:current-tripleo
         networkAttachment: internalapi
   ovs:
     template:
       ovsContainerImage: "quay.io/skaplons/ovs:latest"
```

Comment:
I could not find the webhook logic to define defaults for this so am leaving it now.

Reply:
Ignore this. I noticed this also have the default set by webhook.